thirdparty/CMakeLists.txt

```diff
@@ -178,25 +169,60 @@ if(ENABLE_ONEDNN_FOR_GPU)
                        "-DDNNL_LIBRARY_TYPE=STATIC"
                        "-DOpenCL_LIBRARY=${OpenCL_LIBRARY}"
                        "-DOpenCL_INCLUDE_DIR=${OpenCL_INCLUDE_DIR}"
-                       "-DOPENCL_VERSION_2_2=${OPENCL_VERSION_2_2}"
-            BUILD_COMMAND ${CMAKE_COMMAND} --build . --config ${CMAKE_BUILD_TYPE} --target onednn_gpu --parallel ${CORES_COUNT}
```

Comment:
why not just to use `$<CONFIG>` here instead of `${CMAKE_BUILD_TYPE}`?

Reply:
See https://gitlab.kitware.com/cmake/cmake/-/issues/17645

Reply:
What do you mean?
$<CONFIG> is used in multi configuration case. Since I have to use two different commands anyway (to workaround CMake limitations), I just left the original command for other branch unchanged.